### PR TITLE
added user endpoints

### PR DESCRIPTION
**Pull request by tailrin**

### CODE REVIEW

File: src/users/users-service.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      .into('users')
+      .returning('*')
+      .then(([user]) => user)
+  },
+  validatePassword(password) {
+    if (password.length < 8) {
+      return 'Password be longer than 8 characters'
+    }
+    if (password.length > 72) {
+      return 'Password be less than 72 characters'
```
Comment (aaoyekanmi4) [inline src/users/users-service.js:25]: "Passwords _must_ be longer than 8 characters' 
"Passwords _must_ be less than 8 characters'

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  PORT: process.env.PORT || 8000,
+  NODE_ENV: process.env.NODE_ENV || 'development',
+  DB_URL: process.env.DATABASE_URL || 'postgresql://thingful@localhost/bug-trapper',
+  TEST_DB_URL: process.env.TEST_DATABASE_URL ||'postgresql://thingful@localhost/bug-trapper-test',
```
Comment (aaoyekanmi4) [inline src/config.js:5]: Let's change these names for the current project

File: src/app.js [hunk 3]
```diff
@@ -14,6 +17,9 @@ const morganOption = (NODE_ENV === 'production')
 app.use(morgan(morganOption))
 app.use(helmet())
 app.use(cors())
+app.use('/api/auth', authRouter)
+app.use('/api/users', usersRouter)
+app.use('/company', companyRouter)
```
Comment (aaoyekanmi4) [inline src/app.js:22]: (Optional) do we want to keep all routes /api/{variable} or just the user ones?

Comment (tailrin) [reply src/app.js:22]: All routes are necessary.

File: src/config.js (hunk 2 above)
Comment (tailrin) [reply src/config.js:5]: Why do we want to change these name?

File: src/users/users-service.js (hunk 1 above)
Comment (tailrin) [reply src/users/users-service.js]: change has been mande

File: src/app.js (hunk 3 above)
Comment (tailrin) [reply src/app.js]: change has been made

File: src/config.js (hunk 2 above)
Comment (LZylstra) [reply src/config.js:5]: My branch actually has a config file with a different database name

**Pull request merged**